data/p10/attribute_types_obmc.xml

```diff
@@ -751,6 +751,50 @@
         </simpleType>
     </attribute>
 
+    <enumerationType>
+        <description>
+            The functional state of the system oscillators.
+        </description>
+        <enumerator>
+            <name>NO_DECONFIG</name>
+            <value>0x0</value>
+        </enumerator>
+        <enumerator>
+            <name>A_DECONFIG</name>
```

Comment:
What do you mean by A_DECONFIG and B_CONFIG? It will also be good to add some details about this attribute usage on the BMC side.

Reply:
A and B are clock positions. Currently 2 clock's supported. 

Commit message already provides the details of the flow , which inlcudes BMC side flow.

Reply:
"A" corresponds to the CP_REFCLOCK_SELECT value of OSC0 and "B" to OSC1. So A_DECONFIG means OSC1 is active, B_DECONFIG means OSC0 is active.

Reply:
> > A and B are clock positions
> 
> Ok, can you please mention that details in attribute description tag? How we are assuming the clock positions with this value it might vary from hardware perspective or physical path value, right? A represent 0, B represent 1 so on? If you have some relevant patches please point for the reference.

done